docs/foundation/js/logobox.js

```diff
@@ -1,45 +1,50 @@
-import React from 'react';
+import React from "react";
 
 class LogoBox extends React.Component {
   constructor(props) {
-    super(props);
+    super();
     this.state = {
-      theme:     'logo-light', //default
-      logo:       props.light, //default
-      download:   props.download,
-      logoLight:  props.light,
-      logoDark:   props.dark,
-    }
+      theme: "logo-light", //default
+      logo: props.light //default
+    };
+    this.download = this.download.bind(this);
+    this.changeTheme = this.changeTheme.bind(this);
   }
   changeTheme() {
     let currentTheme = this.state.theme;
-    console.log('currentTheme', currentTheme, 'logo', this.state.logo)
-    if(currentTheme === 'logo-dark') {
-      this.setState({theme: 'logo-light', logo: this.state.logoLight})
-    }
-    else {
-      this.setState({theme: 'logo-dark', logo: this.state.logoDark})
+    console.log("currentTheme", currentTheme, "logo", this.state.logo);
+    if (currentTheme === "logo-dark") {
+      this.setState({ theme: "logo-light", logo: this.props.light });
+    } else {
+      this.setState({ theme: "logo-dark", logo: this.props.dark });
     }
   }
   download() {
-    console.log('download', this.state.download);
-    window.open(this.state.download);
+    console.log("download", this.props.download);
+    window.open(this.props.download);
   }
   render() {
-    return(
-      <div className="logobox">
+    return (
+      <div className={`logobox h-100 ${this.state.theme}`}>
         <div className="logobox-header">
+          <div className="my-auto mr-auto ml-3">
```

Comment:
`my-auto mr-auto ml-3 ` not require b/c div inside the flex and flex items are center aligned.